code/modules/research/ordnance/tank_compressor.dm

```diff
@@ -229,7 +203,7 @@
 	if(!inserted_tank)
 		return FALSE
 	var/datum/gas_mixture/tank_air = inserted_tank.return_air()
-	if(!tank_air.return_pressure() >= PUMP_MAX_PRESSURE)
+	if(tank_air.return_pressure() >= (PUMP_MAX_PRESSURE + ONE_ATMOSPHERE))
```

Comment:
is this an important target to hit? why is * 26 that much better then * 25

Reply:
Is to give a leeway for the tank pressure so that it can be removed, since the compressor doesn't have a pressure target but a max liters per tick pump

Reply:
leeway?

Reply:
Like is not easy to stop perfectly where you want, this will give them that little pressure gap that might make the difference between being able to take the tank out or not

Reply:
you can't take the tank out?

Reply:
if the pressure is higher than a threshold (that is PUMP_MAX_PRESSURE) you can't eject it

Reply:
Ok sure